process-compose/suzuka-full-node/process-compose.gas-dos.yml

```diff
@@ -0,0 +1,16 @@
+version: "3"
+
+environment:
+
+processes:
+
+  gas-dos-test:
```

Comment:
Do we really need an overlay for each small test area? The test run could be rolled into `process-compose.test.yml` since it needs the same process-compose environment.

Reply:
I personally prefer it for targeting different tests. One thing I was thinking is that we could update the scripts to have a `/test` directory where it reads additional overlays. It might help with the organization.